lib/index.js

```diff
@@ -1,54 +1,70 @@
-exports.internal.Settings = exports.Settings;
+export { default as Serializer } from './serializer/index';
+
+export { default as Math } from './common/Math';
+export { default as Vec2 } from './common/Vec2';
+export { default as Vec3 } from './common/Vec3';
+export { default as Mat22 } from './common/Mat22';
+export { default as Mat33 } from './common/Mat33';
+export { default as Transform } from './common/Transform';
+export { default as Rot } from './common/Rot';
+
+export { default as AABB } from './collision/AABB';
+
+export { default as Shape } from './Shape';
+export { default as Fixture } from './Fixture';
+export { default as Body } from './Body';
+export { default as Contact } from './Contact';
+export { default as Joint } from './Joint';
+export { default as World } from './World';
+
+export { default as Circle } from './shape/CircleShape';
+export { default as Edge } from './shape/EdgeShape';
+export { default as Polygon } from './shape/PolygonShape';
+export { default as Chain } from './shape/ChainShape';
+export { default as Box } from './shape/BoxShape';
+
+export { CollideCircles } from './shape/CollideCircle';
+export { CollideEdgeCircle } from './shape/CollideEdgeCircle';
+export { CollidePolygons } from './shape/CollidePolygon';
+export { CollidePolygonCircle } from './shape/CollideCirclePolygone';
+export { CollideEdgePolygon } from './shape/CollideEdgePolygon';
+
+export { default as DistanceJoint } from './joint/DistanceJoint';
+export { default as FrictionJoint } from './joint/FrictionJoint';
+export { default as GearJoint } from './joint/GearJoint';
+export { default as MotorJoint } from './joint/MotorJoint';
+export { default as MouseJoint } from './joint/MouseJoint';
+export { default as PrismaticJoint } from './joint/PrismaticJoint';
+export { default as PulleyJoint } from './joint/PulleyJoint';
+export { default as RevoluteJoint } from './joint/RevoluteJoint';
+export { default as RopeJoint } from './joint/RopeJoint';
+export { default as WeldJoint } from './joint/WeldJoint';
+export { default as WheelJoint } from './joint/WheelJoint';
+
+export { default as Settings } from './Settings';
+
+export { default as Sweep } from './common/Sweep';
+export { default as Manifold } from './Manifold';
+export { default as Distance } from './collision/Distance';
+export { default as TimeOfImpact } from './collision/TimeOfImpact';
+export { default as DynamicTree } from './collision/DynamicTree';
+
+
+import { CollidePolygons } from './shape/CollidePolygon';
+import { default as Settings } from './Settings';
+import { default as Sweep } from './common/Sweep';
+import { default as Manifold } from './Manifold';
+import { default as Distance } from './collision/Distance';
+import { default as TimeOfImpact } from './collision/TimeOfImpact';
+import { default as DynamicTree } from './collision/DynamicTree';
+
+/** @deprecated Merged with main namespace */
+export const internal = {
```

Comment:
This my be a Problem on my side, but this doesn't seem to be bundled (planck.internal is undefined in Node and the browser for me)
Or maybe deprecated exports don't get bundled by rollup? Would seem a bit strange though

Reply:
Could you try building it?

Reply:
Whoops! 😂 Works now 😅